### DISCUSSION TITLE
Problem deleting IP Pool

### DISCUSSION THREAD

**Discussion by lvaro22** [postalserver/postal#2173] [Help with using Postal]
Good morning, I'm having a problem deleting an ip pool.

When I go to the section "IP POOLS" on the up right side of the screen, and I want to delete an ip pool previously configurated, the next message appears; "IP pool can not be removed because it is still assigned to server/rules", but I have already removed all assigned rules from the servers.

Previouly, doing tests, I removed all servers and organizations, but the error message was still appearing.

For now, I have to remove it from the database to resolve the problem.

Is this a bug, or am I doing something wrong ?

thanks for your attention.

**Comment by julianorib** [comment]
up
I'm having this same problem.

**Comment by traderpedroso** [comment]
First delete the all IP inside the pool and update the pool! When there is no IP inside the pool, you can delete it normal!
